### DISCUSSION TITLE
Add ActivityPub support

### DISCUSSION THREAD

**Discussion by jeherve** [microfeed/microfeed#34] [Ideas]
This is a neat little project, thank you! I appreciate that you added an RSS feed in the first version of microfeed. It is very important for podcasters and bloggers alike.

Would you consider adding ActivityPub support to the project, so anyone could follow a microfeed site on the fediverse?

Here are some useful links that may help you get started, although you may find other useful resources on the web about this, since more and more people are considering ActivityPub for their projects.

- https://activitypub.rocks/
- https://paul.kinlan.me/adding-activity-pub-to-your-static-site/

Thanks for the consideration!

**Comment by wenbinf** [comment]
Thanks, @jeherve !

Yes, ActivityPub is on our roadmap :)

**Reply by jeherve** [reply]
Excellent, glad to hear it!

**Reply by amblerkr** [reply]
I hate X. I'm waiting for this.

**Reply by wenbinf** [reply]
@amblerkr - how will you use ActivityPub via microfeed? do you have a specific use case in mind?

**Reply by amblerkr** [reply]
Please wait while I write a new post.
